rop.spec

```diff
@@ -0,0 +1,56 @@
+Name:      rop
+Version:   0.0.1
+Release:   0%{?dist}
+Summary:   Install Foreman using containers
+
+License:   GPL-2-only
+URL:       https://github.com/theforeman/foreman-quadlet
+Source:    https://github.com/theforeman/foreman-quadlet/releases/download/%{version}/%{name}-%{version}.tar.gz
+
+BuildArch: noarch
+Requires:  ansible-collection-ansible-posix
+Requires:  ansible-collection-community-crypto
+Requires:  ansible-collection-community-general
+Requires:  ansible-collection-community-postgresql
+Requires:  ansible-collection-containers-podman >= 1.14.0
+Requires:  ansible-collection-theforeman-foreman
+Requires:  ansible-collection-theforeman-operations
+Requires:  python3-obsah >= 1.1
+
+# These are needed on the target host, which is usually localhost
+Recommends:  podman
+Recommends:  python3-libsemanage
+Recommends:  python3-psycopg2
+
+# Tab completion is nice to have
+Suggests:  bash-completion
+Suggests:  python3-argcomplete
+
+%description
+Install Foreman using containers. They are deployed as podman quadlets.
+
+%prep
+%setup -q -n %{name}-%{version}
+
+%build
+cat > inventories/quadlet <<INVENTORY
+[quadlet]
+localhost ansible_connection=local
+INVENTORY
+
+sed -i '/^OBSAH_BASE=/ s|=.\+|=%{_datadir}/%{name}|' rop
+
+%install
+install -d -m0755 %{buildroot}%{_datadir}/%{name}
+install -d -m0755 %{buildroot}%{_bindir}
+
+cp -r inventories src %{buildroot}%{_datadir}/%{name}
```

Comment:
IMHO the inventory should live in `/etc` as it's something users might want to edit (ext db at some point?)

Reply:
I thought to keep it simple now, but we should discuss the multi host story.

If you take it to the extreme, would you also deploy new (content) proxies with it?

Reply:
I tend to agree on putting it in etc. Or we ship this default one, and we support `/etc` as well where `/etc` has precedence. This could lead to unexpected behaviour though if we have multiple potential inventory sources of truth.

Reply:
new content proxies, if they are deployed by a new (non existent) installation method, yes

new content proxies, deployed by the classical method… maybe

Reply:
```suggestion
cp -r src %{buildroot}%{_datadir}/%{name}
cp inventories/quadlet %{buildroot}%{_sysconfdir}/%{name}/inventory
```